.eslintrc

```diff
@@ -1,12 +1,13 @@
 {
   "extends": ["@grafana/eslint-config"],
   "root": true,
-  "plugins": ["no-only-tests", "@emotion", "lodash"],
+  "plugins": ["no-only-tests", "@emotion", "lodash", "jest"],
   "rules": {
     "no-only-tests/no-only-tests": "error",
     "react/prop-types": "off",
     "@emotion/jsx-import": "error",
-    "lodash/import-scope": [2, "member"]
+    "lodash/import-scope": [2, "member"],
+    "jest/no-focused-tests": "error"
```

Comment:
Does this also handle `it.only` maybe we can remove the `no-only-tests` plugin in that case?

Reply:
You're right, it does cover both use cases. I'll remove the other plugin.